test/HHConfiguration.py

```diff
@@ -71,40 +71,34 @@
+                    IsoMu8orIsoTkMu8leg = cms.untracked.FileInPath('cp3_llbb/HHAnalysis/data/Efficiencies/Muon_DoubleIsoMu17TkMu8_IsoMu8legORTkMu8leg.json'),
 
-                    DoubleEleHighPtleg = cms.untracked.FileInPath('cp3_llbb/Framework/data/Efficiencies/Electron_HLT_DoubleEleLegHigPt_HWW.json'),
-                    DoubleEleLowPtleg = cms.untracked.FileInPath('cp3_llbb/Framework/data/Efficiencies/Electron_HLT_DoubleEleLegLowPt_HWW.json'),
+                    DoubleEleHighPtleg = cms.untracked.FileInPath('cp3_llbb/HHAnalysis/data/Efficiencies/Electron_IsoEle23Leg.json'),
+                    DoubleEleLowPtleg = cms.untracked.FileInPath('cp3_llbb/HHAnalysis/data/Efficiencies/Electron_IsoEle12Leg.json'),
 
-                    EleMuHighPtleg = cms.untracked.FileInPath('cp3_llbb/Framework/data/Efficiencies/Electron_HLT_EleMuLegHigPt_HWW.json'),
-                    MuEleLowPtleg = cms.untracked.FileInPath('cp3_llbb/Framework/data/Efficiencies/Electron_HLT_MuEleLegLowPt_HWW.json'),
+                    EleMuHighPtleg = cms.untracked.FileInPath('cp3_llbb/HHAnalysis/data/Efficiencies/Electron_IsoEle23Leg.json'),
+                    MuEleLowPtleg = cms.untracked.FileInPath('cp3_llbb/HHAnalysis/data/Efficiencies/Electron_IsoEle12Leg.json'),
 
-                    IsoMu8leg = cms.untracked.FileInPath('cp3_llbb/Framework/data/Efficiencies/Muon_DoubleMu_IsoMu8leg_Run2016_PTvsETA_Run271036to276811_HWW_weighted.json'),
-                    IsoMu23leg = cms.untracked.FileInPath('cp3_llbb/Framework/data/Efficiencies/Muon_DoubleMu_IsoMu23leg_Run2016_PTvsETA_Run271036to276811_HWW_weighted.json'),
+                    IsoMu8leg = cms.untracked.FileInPath('cp3_llbb/HHAnalysis/data/Efficiencies/Muon_XPathIsoMu8leg.json'),
+                    IsoMu23leg = cms.untracked.FileInPath('cp3_llbb/HHAnalysis/data/Efficiencies/Muon_XPathIsoMu23leg.json'),
             )
         )
     )
 )
 
-# Add PUPPI MET
-puppiCfg = cms.PSet(METProducer.default_configuration.clone())
-puppiCfg.prefix = cms.string('puppimet_')
-puppiCfg.parameters.met = cms.untracked.InputTag('slimmedMETsPuppi')
-framework.addProducer('puppimet', puppiCfg)
-
 # Remove fat jets
 framework.removeProducer('fat_jets')
 
 framework.getProducer('hlt').parameters.triggers = cms.untracked.FileInPath('cp3_llbb/HHAnalysis/data/triggers.xml')
-#framework.getProducer('jets').parameters.cut = cms.untracked.string("pt > 20")
+# framework.getProducer('jets').parameters.cut = cms.untracked.string("pt > 20")
 #framework.getProducer('jets').parameters.computeRegression = cms.untracked.bool(True)
 
 framework.redoJEC()
 
-framework.smearJets()
-
-framework.doSystematics(['jec', 'jer'])
+if not runOnData:
+    framework.smearJets(resolutionFile='cp3_llbb/Framework/data/Spring16_25nsV10_MC_PtResolution_AK4PFchs.txt', scaleFactorFile='cp3_llbb/Framework/data/Spring16_25nsV10_MC_SF_AK4PFchs.txt')
```

Comment:
It seems it was updated both in data and MC: https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2804.html